core/src/backends/eclair/rest/node.rs

```diff
@@ -14,22 +14,19 @@ pub struct EclairRest {
 }
 
 impl EclairRest {
-    pub fn new(config: NodeConfig) -> Result<Self, Error> {
-        let config: EclairRestConfig = config.into();
-
-        let encoded_auth = format!("{}:{}", config.username, config.password);
-        let auhtorization = format!("Basic {}", base64::encode(encoded_auth));
+    pub fn new(config: EclairRestConfig) -> Result<Self, Error> {
+        let auth = format!("{}:{}", &config.username, &config.password);
+        let authorization = format!("Basic {}", base64::encode(encoded_auth));
```

Comment:
```suggestion
        let authorization = format!("Basic {}", base64::encode(auth));
```
Variable name edited in e55e0e4fb20d951fd6079d2166eb18579eca5344.